dbt_subprojects/hourly_spellbook/models/_sector/perpetual/projects/gains_network/gains_network_v1_base_perpetual_trades.sql

```diff
@@ -0,0 +1,198 @@
+{{ config(
+    schema = 'gains_network_base',
+    alias = 'perpetual_trades',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
```

Comment:
Since this is an incremental view, we need to apply incremental filters in the related selection.